### PR TITLE
Feature/4.4.0 py version fix bundled

### PR DESCRIPTION
**Pull request by maciekswat**
Fixed compilation on Windows of FieldExtractor.cpp.  Bumped up versions al added a fix for segfault occurring with cell level vector fields due to function __get__ item releasing the GIL. More info here: https://stackoverflow.com/questions/74861186/access-vaiolation-in-pyarray-simplenew

As of now, the cc3d builds and runs on osx and Windows with python 3.8+

### CODE REVIEW

File: cc3d/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from os.path import dirname, join, abspath
 from pathlib import Path
 
-__version__ = "4.4.0"
+__version__ = "4.5.0"
```
Comment (tjsego) [inline cc3d/__init__.py:12]: How about 4.4.1, since this is a patch release?

File: CompuCell3D/CMakeLists.txt [hunk 2]
```diff
@@ -70,7 +70,7 @@ SET(CC3D_PYTHON_MODULE_DIR ${CMAKE_SOURCE_DIR}/../)
 
 set(COMPUCELL3D_MAJOR_VERSION 4)
 set(COMPUCELL3D_MINOR_VERSION 4)
-set(COMPUCELL3D_BUILD_VERSION 0)
+set(COMPUCELL3D_BUILD_VERSION 1)
```
Comment (tjsego) [inline CompuCell3D/CMakeLists.txt:73]: 4.4.1 or 4.5.0?

File: CompuCell3D/ReleaseNotes.rst [hunk 3]
```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+
+Version 4.4.0-py-version-fix
```
Comment (tjsego) [inline CompuCell3D/ReleaseNotes.rst:5]: Needs updated

File: cc3d/__init__.py [hunk 4]
```diff
@@ -9,11 +9,12 @@
 from os.path import dirname, join, abspath
 from pathlib import Path
 
-__version__ = "4.4.0"
+__version__ = "4.4.1"
```
Comment (tjsego) [inline cc3d/__init__.py:12]: 4.4.1 or 4.5.0?

File: conda-recipes-compucell3d/versions.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.5.0
```
Comment (tjsego) [inline conda-recipes-compucell3d/versions.yaml:1]: 4.4.1 or 4.5.0?

File: conda-recipes/versions.yaml [hunk 6]
```diff
@@ -1,8 +1,8 @@
-version: 4.4.0
+version: 4.4.1
```
Comment (tjsego) [inline conda-recipes/versions.yaml:1]: 4.4.1 or 4.5.0?

**Review by tjsego — CHANGES_REQUESTED** [review]
Minor cleanup here. Otherwise, looks great.

File: CompuCell3D/ReleaseNotes.rst (hunk 3 above)
Comment (maciekswat) [reply CompuCell3D/ReleaseNotes.rst]: Done

File: cc3d/__init__.py (hunk 4 above)
Comment (maciekswat) [reply cc3d/__init__.py:12]: Fixed

File: cc3d/__init__.py (hunk 1 above)
Comment (maciekswat) [reply cc3d/__init__.py]: Fixed

File: conda-recipes-compucell3d/versions.yaml (hunk 5 above)
Comment (maciekswat) [reply conda-recipes-compucell3d/versions.yaml]: Fixed

File: conda-recipes/versions.yaml (hunk 6 above)
Comment (maciekswat) [reply conda-recipes/versions.yaml:1]: Fixed

**Review by tjsego — APPROVED** [review]

**Pull request merged**